packages/angular/projects/clr-angular/src/data/datagrid/datagrid.spec.ts

```diff
@@ -673,6 +718,43 @@ export default function (): void {
         const rows = this.context.clarityElement.querySelectorAll('.datagrid-cell');
         expect(['1', '1', '2', '4', '3', '9']).toEqual([...rows].map(r => r.textContent));
       });
+
+      it('sorting by column works when pagination is also added', function () {
```

Comment:
nit: The issue of this bug is not the pagination but the re-rendering the rows/columns - so the description must be just `sorting is working - yahooo` 

What I'm saying is that not only the pagination will trigger this.